com.hazelcast.jet.impl.util.ReflectionUtilsTest.when_nestedClassesOf_then_returnsAllNestedClasses

_master_ (commit 83371a644146617653c1e5a83c72a04264b4175c)

Failed on openj9-11: https://jenkins.hazelcast.com/job/Hazelcast-master-Openj9-11/121/testReport/junit/com.hazelcast.jet.impl.util/ReflectionUtilsTest/when_nestedClassesOf_then_returnsAllNestedClasses/
<details><summary>Stacktrace:</summary>

```
io.github.classgraph.ClassGraphException: Uncaught exception during scan
	at io.github.classgraph.ClassGraph.scan(ClassGraph.java:1606)
	at io.github.classgraph.ClassGraph.scan(ClassGraph.java:1623)
	at io.github.classgraph.ClassGraph.scan(ClassGraph.java:1636)
	at com.hazelcast.jet.impl.util.ReflectionUtils.nestedClassesOf(ReflectionUtils.java:232)
	at com.hazelcast.jet.impl.util.ReflectionUtilsTest.when_nestedClassesOf_then_returnsAllNestedClasses(ReflectionUtilsTest.java:172)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:115)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:107)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: java.lang.RuntimeException: Cannot use reflection after ScanResult has been closed
	at nonapi.io.github.classgraph.reflection.ReflectionUtils.invokeMethod(ReflectionUtils.java:241)
	at io.github.classgraph.ModuleReaderProxy.<init>(ModuleReaderProxy.java:69)
	at io.github.classgraph.ModuleRef.open(ModuleRef.java:310)
	at nonapi.io.github.classgraph.fastzipfilereader.NestedJarHandler$5$1.newInstance(NestedJarHandler.java:356)
	at nonapi.io.github.classgraph.fastzipfilereader.NestedJarHandler$5$1.newInstance(NestedJarHandler.java:353)
	at nonapi.io.github.classgraph.recycler.Recycler.acquire(Recycler.java:79)
	at nonapi.io.github.classgraph.recycler.Recycler.acquireRecycleOnClose(Recycler.java:101)
	at io.github.classgraph.ClasspathElementModule.scanPaths(ClasspathElementModule.java:300)
	at io.github.classgraph.Scanner$4.processWorkUnit(Scanner.java:1095)
	at io.github.classgraph.Scanner$4.processWorkUnit(Scanner.java:1089)
	at nonapi.io.github.classgraph.concurrency.WorkQueue.runWorkLoop(WorkQueue.java:246)
	at nonapi.io.github.classgraph.concurrency.WorkQueue.runWorkQueue(WorkQueue.java:161)
	at io.github.classgraph.Scanner.processWorkUnits(Scanner.java:314)
	at io.github.classgraph.Scanner.openClasspathElementsThenScan(Scanner.java:1087)
	at io.github.classgraph.Scanner.call(Scanner.java:1146)
	at io.github.classgraph.Scanner.call(Scanner.java:81)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	... 1 more

```
</details>


<details><summary>Standard output:</summary>

```
Finished Running Test: when_findPropertySetter_publicStatic_then_returnsNull in 0.003 seconds.

```
</details>

Standard output can be found here - https://s3.console.aws.amazon.com/s3/buckets/j-artifacts/Hazelcast-master-Openj9-11/121/

--- a/pom.xml
+++ b/pom.xml
@@ -94,7 +94,7 @@
         <antlr4.version>4.9.3</antlr4.version>
         <avro.version>1.11.1</avro.version>
         <aws.sdk.version>1.12.414</aws.sdk.version>
-        <classgraph.version>4.8.155</classgraph.version>
+        <classgraph.version>4.8.156</classgraph.version>
         <debezium.version>1.9.7.Final</debezium.version>
         <grpc.version>1.48.0</grpc.version>
         <guava.version>31.1-jre</guava.version>

